extensions/amp-timeago/0.1/locales.js

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * This file registers all necessary locales supported by amp-timeago.
+ */
+
+import {format, register} from 'timeago.js/dist/timeago.full.min.js';
```

Comment:
For this to be included by the AMP build, I'm pretty sure the `timeago` dependency will need to be added to this list:

https://github.com/ampproject/amphtml/blob/3d0c39b3a45878dfc029f42c67a0782633440bcf/build-system/compile/sources.js#L40-L68